manifests/crd/odhdashboardconfigs.opendatahub.io.crd.yaml

```diff
@@ -119,11 +119,7 @@ spec:
                                 type: string
                 notebookController:
                   type: object
-                  required:
-                    - enabled
                   properties:
-                    enabled:
-                      type: boolean
```

Comment:
This is a problem as you noted -- it's important to be very careful about changing our Dashboard Config in backwards incompatible ways. Don't remove things without proper consideration to the cost of removing them.

Resources are reconciled by the operator, they cannot "remove" things cleanly by deleting the resource. We need to make sure we document this. More for https://github.com/opendatahub-io/odh-dashboard/issues/1611 to improve.

Reply:
There are other places where that setting from crd is evaluated and removed in the files of this PR.